Sources/protoc-gen-swift/OneofGenerator.swift

```diff
@@ -345,6 +345,7 @@ class OneofGenerator {
               hadValueTest = "hadOneofValue"
           } else {
               p.print("var v: \(field.swiftType)?")
+              p.print("if case \(field.dottedSwiftName)(let _v) = \(storedProperty) {v = _v}")
```

Comment:
Can you split this change out, since this is the actual generator change, it makes it much easier to see.